static/defaults.config.php

```diff
@@ -237,6 +237,10 @@
 		// Periodically delete waiting database processes.
 		'delete_sleeping_processes' => false,
 
+		// delete-blocked-servers (Boolean)
+		// Delete blocked servers if possible.
```

Comment:
Please expand on the « if possible » aspect.

Reply:
Done